GameServ/GameHost.cpp

```diff
@@ -484,6 +504,8 @@ void dm_read_sdl(GameHost_Private* host, GameClient_Private* client,
         host->m_localState.add(update);
         host->m_ageSdlHook.add(update);
```

Comment:
Remind me, what's the difference between these two?  And why are we saving one and broadcasting the other?  >.>